grpc/src/test/java/io/stargate/grpc/service/QueryParametersTest.java

```diff
@@ -81,37 +81,40 @@ public void queryParameters(QueryParameters actual, Parameters expected) {
 
   public static Stream<Arguments> queryParameterValues() {
     return Stream.of(
-        arguments(cqlQueryParameters().build(), Parameters.builder().build()),
+        arguments(cqlQueryParameters().build(), Parameters.builder().pageSize(100).build()),
```

Comment:
Should this refer to the constant (Service.DEFAULT_PAGE_SIZE) in case that changes?